modules/campaigns/module.php

```diff
@@ -0,0 +1,196 @@
+	}
+
+	public function get_taxonomy_page_slug( $add_post_type_filter = true ) {
+		$slug = 'edit-tags.php?taxonomy=' . self::TAXONOMY_NAME;
+
+		if ( $add_post_type_filter ) {
+			return $slug . '&post_type=page';
+		}
+
+		return $slug;
+	}
+
+	public function get_menu_slug() {
+		return $this->campaigns_menu_slug;
+	}
+
+	public function get_taxonomy_terms() {
+		return $this->terms;
+	}
+
+	private function register_campaigns_taxonomy() {
+		$taxonomy_labels = [
+			'name' => _x( 'Campaigns', 'Campaigns taxonomy general name', 'elementor' ),
+			'singular_name' => _x( 'Campaign', 'Campaign singular name', 'elementor' ),
+			'search_items' => __( 'Search Campaigns', 'elementor' ),
+			'popular_items' => __( 'Popular Campaigns', 'elementor' ),
+			'all_items' => __( 'All Campaigns', 'elementor' ),
+			'edit_item' => __( 'Edit Campaign', 'elementor' ),
+			'update_item' => __( 'Update Campaign', 'elementor' ),
+			'add_new_item' => __( 'Add New Campaign', 'elementor' ),
+			'new_item_name' => __( 'New Campaign Name', 'elementor' ),
+			'separate_items_with_commas' => __( 'Separate Campaigns with commas', 'elementor' ),
+			'add_or_remove_items' => __( 'Add or remove Campaigns', 'elementor' ),
+			'choose_from_most_used' => __( 'Choose from the most used Campaigns', 'elementor' ),
+			'not_found' => __( 'No Campaigns found.', 'elementor' ),
+			'menu_name' => __( 'Campaigns', 'elementor' ),
+		];
+
+		$taxonomy_args = [
+			'labels' => $taxonomy_labels,
+			'show_in_menu' => false,
+			'meta_box_cb' => false, // Don't show metabox.
+			'show_in_quick_edit' => false,
+			'query_var' => is_admin(),
+			'rewrite' => false,
+		];
+
+		// Register taxonomy, associate it with Landing Pages and Popups (Actually 'elementor_library', not just Popups).
+		// Landing pages' CPT is just 'page'.
+		register_taxonomy( self::TAXONOMY_NAME, [ 'page', Source_Local::CPT ], $taxonomy_args );
+	}
+
+	/**
+	 * Add Menu Page
+	 *
+	 * Runs on WP's 'admin_menu' hook. Registers the 'Campaigns' admin menu item.
+	 *
+	 * @since 3.1.0
+	 */
+	private function add_menu_page() {
+		// If there are no campaigns, show the 'Create Your First Campaign' page.
+		// Otherwise, show the campaigns editing screen.
+		if ( empty( $this->terms ) ) {
+			$campaigns_page = [ $this, 'get_empty_campaigns_page' ];
+		} else {
+			$campaigns_page = '';
+		}
+
+		$campaigns_title = __( 'Campaigns', 'elementor' );
+
+		// Add the Campaigns Menu Item.
+		add_menu_page(
+			$campaigns_title,
+			$campaigns_title,
+			'manage_options',
+			$this->campaigns_menu_slug,
+			$campaigns_page,
+			'dashicons-filter',
+			// Other plugin's menu positions are also 58.x. The .7 is to appear after Elementor's 'Templates' menu item.
+			'58.7'
+		);
+	}
+
+	/**
+	 * Get Empty Campaigns Page
+	 *
+	 * Returns the HTML output that is displayed in the 'Campaigns' page when there are no campaigns (terms) registered.
+	 *
+	 * @since 3.1.0
+	 */
+	public function get_empty_campaigns_page() {
```

Comment:
It seems like this method doesn't return any value. Therefore, its name shouldn't be started with a `get`. Fix the doc as well.

Reply:
Done.